release-notes-agent/src/main/resources/application.properties

```diff
@@ -21,8 +21,16 @@ jira.auth.token=${JIRA_TOKEN}
 jira.version.custom.field.id=
 jira.github.custom.field.id=
 
+
 release.notes.agent.admin.email=
 release.notes.agent.subscribers=
+release.notes.agent.path.to.system.folder.with.templates=
+release.notes.agent.classpath.to.templates=
+release.notes.agent.type.of.template.files=
```

Comment:
do we actually need this at all? can we store html template in files without any extension? I think yes, since after all we just fill it and put to email body